cli/cli.go

```diff
@@ -130,5 +133,23 @@ func HandleRelayFlags(relayCmd *kingpin.CmdClause, opts *Options) {
 		Default(DefaultHTTPListenAddress).
 		Envar("PLUMBER_RELAY_HTTP_LISTEN_ADDRESS").
 		StringVar(&opts.RelayHTTPListenAddress)
+}
+
+func ValidateProtobufOptions(dirs []string, rootMessage string) error {
+	if len(dirs) == 0 {
+		return errors.New("at least one '--protobuf-dir' must be set when type " +
+			"is set to 'protobuf'")
+	}
+
+	if rootMessage == "" {
+		return errors.New("'--protobuf-root-message' must be when " +
+			"type is set to 'protobuf'")
+	}
```

Comment:
Let's change the message to "--protobuf-root-message required when type is set to 'protobuf'"